docs/run.js

```diff
@@ -91,11 +112,11 @@ let featureSelectListener = null;
  * Creates the score overlay and draws the table
  *
  * @param {google.maps.Map} map main map
- * @param {Promise<Array<number>>} initialTogglesValuesPromise promise that
- *     returns the poverty and damage thresholds and the poverty weight (from
- *     which the damage weight is derived).
+ * @param {Promise<ScoreComputationParameters>} scoreParametersPromise Will
+ *     resolve once score asset name is known and Firestore fetch done
+ * @return {Promise<void>} Promise that is done when score layer all done
```

Comment:
can you specify all done with what? loading?

Reply:
Expanded.